src/main/java/org/jabref/gui/CopyEntryToDialogView.java

```diff
@@ -0,0 +1,34 @@
+package org.jabref.gui;
+
+import javafx.scene.control.ButtonBar;
+import javafx.scene.control.ButtonType;
+import javafx.scene.layout.Region;
+
+import org.jabref.gui.util.BaseDialog;
+import org.jabref.logic.l10n.Localization;
+
+public class CopyEntryToDialogView extends FXDialog {
+    ButtonType copySelectedEntryOnly = new ButtonType("Copy selected entry only", ButtonBar.ButtonData.OK_DONE);
+    ButtonType copySelectedEntryWithCrossReferencedEntries = new ButtonType("Copy cross references", ButtonBar.ButtonData.OK_DONE);
+    ButtonType copySelectedEntryWithCrossReferencedEntriesRecursively = new ButtonType("Copy cross references recursively", ButtonBar.ButtonData.OK_DONE);
+
+    public CopyEntryToDialogView() {
+        super(AlertType.CONFIRMATION, Localization.lang("Copy entry to library"), true);
+        setHeaderText(null);
+        getDialogPane().setMinHeight(170);
+        setResizable(true);
+        setContentText("""
+                The selected entry cross references 4 other entries.
```

Comment:
Unfortunately, Localization.lang does not work with textblocks yet.... You need to use \n

Reply:
I remember that I tried to switch to [JavaParser](https://github.com/javaparser/javaparser) or similar tooling for better localization handling. However, this did not work in all cases and lead to upstream issues.

I was thinking to use [Kilt](https://poiu-de.github.io/kilt/) for JabRef. However, I did not see fast enough how to keep our localization checks. (Because I like that we have a check that the messages are verbatim included in the code). - And it seems, we would have to change all our parameterized strings (https://github.com/poiu-de/kilt/issues/23). ( `%0` to `{0}`). And some layer on top of Kilt (which is not that harmful, but effort. And thus, the effort added too much for me).

Refs https://github.com/JabRef/jabref/pull/7279

Reply:
Found the implementation: [`c6c7edd` (#8050)](https://github.com/JabRef/jabref/pull/8050/commits/c6c7edd0ffec9d3c5d73d277de851e69912e3361)